### PR TITLE
Basic-server-rendering App Generated v34

### PR DESCRIPTION
**Pull request by robwise**

### CODE REVIEW

File: tester_app/client/app/bundles/HelloWorld/components/HelloWorldWidget.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  };
+
+  // React will automatically provide us with the event `e`
+  _handleChange(e) {
+    const name = e.target.value;
+    this.props.updateName(name);
+  }
+
+  render() {
+    const _handleChange = this._handleChange;
```
Comment (justin808) [inline tester_app/client/app/bundles/HelloWorld/components/HelloWorldWidget.jsx:27]: inline this -- no need to define local var

File: tester_app/client/app/bundles/HelloWorld/containers/HelloWorld.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    _.bindAll(this, "updateName");
+  }
+
+  static propTypes = {
+    name: PropTypes.string.isRequired, // this is passed from the Rails view
+  }
+
+  state = {name: this.props.name} // how to set initial state in es2015 class syntax
+
+  updateName(name) {
```
Comment (justin808) [inline tester_app/client/app/bundles/HelloWorld/containers/HelloWorld.jsx:22]: name starting with underscore

there should be some comment on how we use to differentiate from react methods

this also avoids overriding in base class

File: tester_app/client/app/bundles/HelloWorld/components/HelloWorldWidget.jsx (hunk 1 above)
Comment (robwise) [inline tester_app/client/app/bundles/HelloWorld/components/HelloWorldWidget.jsx:27]: @justin808 you told me to do it Alex's way where he just does it no matter what: https://github.com/shakacode/react_on_rails-generator-results-pre-0/pull/6#discussion_r43587922

File: tester_app/client/app/bundles/HelloWorld/containers/HelloWorld.jsx (hunk 2 above)
Comment (robwise) [inline tester_app/client/app/bundles/HelloWorld/containers/HelloWorld.jsx:22]: @justin808  even in smart components? what about in redux? We don't seem to be doing it in F&G

**Pull request closed without merging**